### PR TITLE
Add proptypes and fix table

### PR DESCRIPTION
**Pull request by VasiliyPolyakov**

### CODE REVIEW

File: src/components/table/table.js [hunk 1]
```diff
@@ -4,18 +4,19 @@ import AmazonLink from './../amazon-link/amazon-link';
...
-    const {products} =  this.props;
-    console.log('products', products);
+    const {products, tableName} =  this.props;
     const headerData = products && products[0].table_features;
 
     return (
       <div className="table__container">
         <div className="table__wrapper">
           <table className="table">
+            <caption className="table__name">{tableName || null}</caption>
```
Comment (mkozhukharenko) [inline src/components/table/table.js]: Please, use h2 for it

And add it to Table of conten component

File: src/components/table/table.scss [hunk 2]
```diff
@@ -8,6 +8,10 @@
   border: 1px solid $text-grey-light;
   position: relative;
 
+  &__name {
```
Comment (mkozhukharenko) [inline src/components/table/table.scss]: name -> title

File: static/admin/config.yml [hunk 3]
```diff
@@ -67,6 +67,7 @@ collections:
         widget: "object"
         fields:
           - {label: "Table present?", name: "isTable", widget: "boolean", default: false}
+          - {label: "Table name", name: "tableName", widget: "string"}
```
Comment (mkozhukharenko) [inline static/admin/config.yml]: table title , please

File: src/components/table/table.js (hunk 1 above)
Comment (VasiliyPolyakov) [reply src/components/table/table.js]: Why do I need to change tag? https://www.w3schools.com/tags/tag_caption.asp

File: static/admin/config.yml (hunk 3 above)
Comment (VasiliyPolyakov) [reply static/admin/config.yml]: done

File: src/components/table/table.scss (hunk 2 above)
Comment (VasiliyPolyakov) [reply src/components/table/table.scss]: done

**Pull request merged**